grid.cpp

```diff
@@ -376,9 +380,12 @@ void initializeGrids(
          calculateCellMoments(mpiGrid[cells[i]], true, true);
       }
    }
-   
+
+   // recompute coupling
+   computeCoupling(mpiGrid, cells, momentsGrid, onDccrgMapRemoteProcessGlobal, onFsgridMapRemoteProcessGlobal, onFsgridMapCellsGlobal);
```

Comment:
This call is not needed, nothing changed the load balance,/task layout/coupling between the previous call and this one.